src/tpm2tss-genkey.c

```diff
@@ -328,6 +378,10 @@ main(int argc, char **argv)
     case TPM2_ALG_ECDSA:
         tpm2Data = genkey_ecdsa();
         break;
+    case TPM2_ALG_AES:
+    case TPM2_ALG_CAMELLIA:
```

Comment:
You sould add a line with `/* FALLTHROUGH */` here so static code analysis knows it is intentional.

Reply:
OK